### ISSUE TITLE
theme broken on Fedora 24 GNOME

### ISSUE THREAD

**Issue by axelmeunier** [hayderctee/masalla-icon-theme#8]
Hey there wonderful theme mate ! Amazing desaturated colors :) 
Just installed it on Fedora 24 with Gnome Shell (fresh install) A lot of icons do not assigned properly by default : like gnome tweak tool, gnome software, libreoffice writer, gedit, cheese, documents etc.

Is it supposed to work ok on Gnome 3.20 ? 

I previously installed it on Ubuntu with zero problems.

On a more positive note, I saw there was no evernote icon so I created one, but i am sure you can make it much better !
[evernote.zip](https://github.com/hayderctee/masalla-icon-theme/files/558566/evernote.zip)

**Comment by hayderctee** [comment]
Hi
It's running well in my machine !! (I try it on gnome shell 3.22.1 on parabola)
download the zip file from here (github) and re-install it .
and thank you for your icon :+1: i will use it in the next version, (the next version 1.0 is almost done) and i will add your name as a contributor :-)

**Comment by axelmeunier** [comment]
Hey thanks for the message ! I reinstalled but same thing. Basically the icon names in the Gnome apps desktop files do not match with yours. Example : cheese is named cheese.svg (or png) in your folder while the desktop file icon name is org.gnome.Cheese... so i copied the icons with new names (only svg !) to match the desktop files. Now it works. I don't really understand... But if anyone stumbles on the same pb i can provide the folder... 
Also it is a pleasure to contribute to such a beautiful icon theme :) cheers

**Comment by hayderctee** [comment]
please try the last version (1.0) , and close this issue if problem fixed

**Comment by axelmeunier** [comment]
Hi Hayder ! I tested the new version. Much better :) Still not working on my system (Fedora 24) : 
gnome software, fbreader, yum extender, authentification, services.
Also I noticed there was no icon for the fedora media writer so I made one, as well as one for Openrefine which I use as a webapp (not many ppl will care about that one i guess).
[icons.zip](https://github.com/hayderctee/masalla-icon-theme/files/590850/icons.zip)
